src/get-is-enabled.js

```diff
@@ -4,13 +4,15 @@ import { find, curry } from 'ramda';
 const matchName = s => x => x.name === s;
 
 // enabled = x: Feature => boolean
-const enabled = x => x && x.enabled ? true : false;
+const enabled = x => (x && x.enabled ? true : false);
 
 // hasDependencies = x: Feature => boolean
-const hasDependencies = x => x.dependencies && x.dependencies.length > 0 ? true : false;
+const hasDependencies = x =>
+  x.dependencies && x.dependencies.length > 0 ? true : false;
```

Comment:
You can delete  `? true : false` here. Comparators always evaluate to `true` or `false` by default.